modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationUtils.java

```diff
@@ -154,17 +152,17 @@ public void lockOperationUpdate(Status status, SyncmlDocument syncmlDocument, De
                     new OperationUtils().updateOperations(syncmlDocument.getHeader().getSource().getLocURI(),
                             pendingDataOperations);
                     try {
-                        NotificationManagementService service = WindowsAPIUtils.getNotificationManagementService();
+                        NotificationManagementService nmService = WindowsAPIUtils.getNotificationManagementService();
                         Notification lockResetNotification = new Notification();
                         lockResetNotification.setOperationId(status.getCommandReference());
                         lockResetNotification.setStatus(String.valueOf(Notification.Status.NEW));
                         lockResetNotification.setDeviceIdentifier(deviceIdentifier);
                         lockResetNotification.setDescription(Constants.SyncMLResponseCodes.LOCKRESET_NOTIFICATION);
 
-                        service.addNotification(lockResetNotification);
+                        nmService.addNotification(lockResetNotification);
                     } catch (NotificationManagementException e) {
                         String msg = "Failure occurred in getting notification service";
```

Comment:
wrong message passing, this should be some error occured during adding notification.
If your expectation is to catch a NPE from this, better avoiding this rather than catching it.